src/person.service.ts

```diff
@@ -0,0 +1,26 @@
+// src/person.service.ts
+import { Injectable, NotFoundException } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+import { Person } from './person.entity';
+import axios from 'axios';
+
+@Injectable()
+export class PersonService {
+  constructor(
+    @InjectRepository(Person)
+    private readonly personRepository: Repository<Person>,
+  ) {}
+
+  async getPersonById(id: number): Promise<any> {
+    const person = await this.personRepository.findOne({ where: { id } });
```

Comment:
Had problems here and asked to fix compiling error:
I asked: 
I have an error: 
src/person.service.ts:16:56 - error TS2559: Type 'number' has no properties in common with type 'FindOneOptions<Person>'.

const person = await this.personRepository.findOne(id);

Suggested solution is const person = await this.personRepository.findOne({ where: { id } });
But I would prefer findOneBy instead